pkg/psalabelsyncer/podsecurity_label_sync_controller.go

```diff
@@ -287,19 +291,19 @@ func (c *PodSecurityAdmissionLabelSynchronizationController) syncNamespace(ctx c
 		} {
 			if ns.Labels[typeLabel] != string(psaLevel) || ns.Labels[versionLabel] != currentPSaVersion {
 				changed = true
-				if nsCopy.Labels == nil {
-					nsCopy.Labels = map[string]string{}
+				if nsApplyConfig.Labels == nil {
+					nsApplyConfig.Labels = map[string]string{}
 				}
 
-				nsCopy.Labels[typeLabel] = string(psaLevel)
-				nsCopy.Labels[versionLabel] = currentPSaVersion
+				nsApplyConfig.Labels[typeLabel] = string(psaLevel)
+				nsApplyConfig.Labels[versionLabel] = currentPSaVersion
 
 			}
 		}
 	}
 
 	if changed {
-		_, err := c.namespaceClient.Update(ctx, nsCopy, metav1.UpdateOptions{})
+		_, err := c.namespaceClient.Apply(ctx, nsApplyConfig, metav1.ApplyOptions{FieldManager: controllerName})
```

Comment:
Now, separately, you've done right by not setting "force".  Now you need to allow for someone else to own these fields.  that means conflict errors are not retried.

So that apierrors.IsConflict() is handled as not an error.